boards/arm/nrf52840dongle_mdk/nrf52840dongle_mdk.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: nrf52840dongle_mdk
+name: nRF52840-Dongle-MDK
+type: mcu
+arch: arm
+ram: 256
+flash: 1024
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - adc
+  - usb_device
+  - usb_cdc
+  - ble
+  - ieee802154
+  - pwm
+  - spi
+  - watchdog
+  - counter
```

Comment:
No newline at end of file

Reply:
Do you mean there is no or new line is not allowed?
Because I can not see any newline at the end.

Reply:
I just cited the warning message that github shows for this file (see it in [this view](https://github.com/zephyrproject-rtos/zephyr/pull/41813/files/b3b23665edca8c87e41f57a2d6a84d35dede1fd9#diff-e2ef73ccd97feb2ea096a969a68abedfa7a5f77f2d1d55780127eee7b9978e48)). You should add newline at the end of this file to get rid of this warning sign. Same story in [Kconfig](https://github.com/zephyrproject-rtos/zephyr/pull/41813/files#diff-35ba2a5d97f1766d493f67415950779310921a9c3fd19b39139452816d20bb12).